src/utils/index.ts

```diff
@@ -810,6 +811,9 @@ export async function getAllCdpsData (blockNumber: number): Promise<Map<string,
       cdps.map(
         async (cdp) => ({
           ...cdp,
+          isDebtsEnoughForLiquidationSpends: (new BigNumber((await getTotalDebt(cdp.asset, cdp.owner)).toString())).div(10**18).gte(
```

Comment:
better to avoid such long oneliners since it is easy to make mistake in it. And also It's harder to understand

Also bigint could be used here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt